frontend/src/metabase-types/guards/click-behavior.ts

```diff
@@ -0,0 +1,77 @@
+import type {
+  BaseActionClickBehavior,
+  DeleteActionClickBehavior,
+  ImplicitActionClickBehavior,
+  InsertActionClickBehavior,
+  UpdateActionClickBehavior,
+} from "metabase-types/api";
+
+const isRecord = (
+  value: unknown,
+): value is Record<string | number | symbol, unknown> => {
+  return typeof value === "object" && value !== null;
+};
+
+const isBaseActionClickBehavior = (
+  value: unknown,
+): value is BaseActionClickBehavior => {
+  return (
+    isRecord(value) &&
+    "type" in value &&
+    value.type === "action" &&
+    "actionType" in value &&
```

Comment:
```suggestion
```

just `typeof value.actionType === "string"` should be enough

Reply:
07022b1eeda80f473ad4b60ebbb80187ef85acdf